test/server_test.rb

```diff
@@ -27,4 +29,25 @@ def test_hello_world_count_increments_by_one_after_every_request
 
     assert_equal expected, response.body
   end
+
+  def test_outputs_formatted_diagnostic
+    client = Hurley::Client.new "http://127.0.0.1:9292"
+    client.header[:accept] = "text/html,application/xhtml+xml,application/xml;" +
```

Comment:
Line is too long. [81/80]<br>Use `\` instead of `+` or `<<` to concatenate those strings.